oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/util.js

```diff
@@ -27,6 +29,17 @@ export function parseFormAppConfig(rootElementId = "form-app") {
   return { rootEl, record, formConfig, recordPermissions, files, links };
 }
 
+export const sanitizeInput = (htmlString, validTags) => {
+  const decodedString = decode(htmlString);
+  const cleanInput = sanitizeHtml(decodedString, {
```

Comment:
does this remove disallowed tags content? Or we need to add `disallowedTagsMode to completelyDiscard` here?